apps/dialtone-documentation/docs/design/sizing/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Sizing
+status: In progress
+description: A system to maintain consistent size and scale.
+---
+
+## Usage
```

Comment:
On both the Size and the Spacing page, let's introduce the difference between them so designers and engineers can understand the difference and why they are distinct.

In the product codebase I too often see them conflated; e.g...
```less
// 👎 Incorrect
padding: var(--dt-size-400)`;

// 👍 Correct
padding: var(--dt-space-400)`;
```

It needs to be clear at this introductory doc level which they should be choose. Below is a _very_ rough example of how it can be introduced. Think about how what I've drafted below can be used in both, perhaps in a visual way: 

```md
## Choosing Size vs Spacing design tokens

### Size
Size design tokens define intrinsic **dimensions** of UI
components. Example CSS properties they correspond to:
- `width`
- `height`
- `border-radius`
- `border-width`
- positioning properties, i.e. `top`, `left`, etc.

### Spacing
Spacing design tokens focus on controlling **spatial 
relationships**; that is, the space between and around
elements. Example CSS properties they correspond to:
- `padding`
- `margin`
- `gap`
```